lucene/queries/src/java/org/apache/lucene/queries/mlt/MoreLikeThis.java

```diff
@@ -92,9 +92,9 @@
  * Reader target = ... // orig source of doc you want to find similarities to
  * Query query = mlt.like( target);
  * 
- * Hits hits = is.search(query);
- * // now the usual iteration thru 'hits' - the only thing to watch for is to make sure
- * //you ignore the doc if it matches your 'target' document, as it should be similar to itself
+ * TopDocs topDocs = is.search(query, 10);
```

Comment:
Could you add documentation around the second parameter? It seems a bit unclear

Reply:
@saschaszott ?

Reply:
added comment